app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -104,19 +197,19 @@ public class RouteModeView : LinearLayout , ViewPager.OnPageChangeListener {
         var lastItemIndex = (pager?.getAdapter() as InstructionAdapter).getCount() - 1
         var itemsUntilLastInstruction = (lastItemIndex - position)
         if(itemsUntilLastInstruction ==  1) {
-            (pager?.getAdapter() as InstructionAdapter)
-                    .setBackgroundColorArrived(findViewByIndex(position + 1))
+            (pager?.getAdapter() as InstructionAdapter).setBackgroundColorArrived(
+                    pager?.findViewWithTag("Instruction_" + (position + 1)))
```

Comment:
You didn't like my helper method `findViewByIndex(int)` :wink: 

Maybe just got clobbered in the rebase?